snowpack/src/commands/build.ts

```diff
@@ -179,7 +184,7 @@ class FileBuilder {
         // Until supported, just exit here.
         if (!resolvedImportUrl) {
           isSuccess = false;
-          logger.error(`${file.locOnDisk} - Could not resolve unkonwn import "${spec}".`);
+          logger.error(`${file.locOnDisk} - Could not resolve unkown import "${spec}".`);
```

Comment:
`unknown`

Reply:
lol I tried to fix a typo and failed 🤦🏻. Goodness